src/main/webapp/dataset.xhtml

```diff
@@ -43,9 +43,9 @@
                 <meta property="og:title" content="#{DatasetPage.title}" />
                 <meta property="og:type" content="article" />
                 <meta property="og:url" content="#{DatasetPage.dataverseSiteUrl}/dataset.xhtml?persistentId=#{dataset.globalId}" />
-                <meta property="og:image" content="#{DatasetPage.dataverseSiteUrl.concat(resource['images/favicondataverse.png'])}" />
```

Comment:
If we're going to delete this file we also need to update integration tests such as this one: https://github.com/IQSS/dataverse/blob/v4.11/src/test/java/edu/harvard/iq/dataverse/api/FilesIT.java#L107

Reply:
As I expected, the integration tests starting failing but I fixed them in pull request #5692